### PR TITLE
lightning: refine description about checksum-via-sql 

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Refine description about checksum-via-sql 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/14518
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -407,6 +407,11 @@ max-allowed-packet = 67_108_864
 # For backward compatibility, bool values "true" and "false" are also allowed for this field.
 # "true" is equivalent to "required" and "false" is equivalent to "off".
 checksum = "required"
+# Specifies whether the ADMIN CHECKSUM TABLE <table> operation is executed via TiDB.
+# The default value is "false", which means that the ADMIN CHECKSUM TABLE <table> command is sent to TiKV for execution via TiDB Lightning.
+# It is recommended that you set this value to "true" to make it easier to locate the problem if checksum fails.
+# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` parameter in TiDB.
+# checksum-via-sql = false
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:414]:
```suggestion
checksum-via-sql = "false"
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -359,8 +359,8 @@ pd-addr = "172.16.31.4:2379"
 # This setting controls the log level of the TiDB library.
 log-level = "error"
 
-# Sets the TiDB session variable to speed up the Checksum and Analyze operations.
-# See https://pingcap.com/docs/dev/reference/performance/statistics/#control-analyze-concurrency
+# Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", you need to execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency = 15` and `checksum-table-concurrency = 2` will not take effect.
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:362]:
```suggestion
# Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", Lightning will execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency = 15` and `checksum-table-concurrency = 2` will not take effect.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -407,6 +407,11 @@ max-allowed-packet = 67_108_864
 # For backward compatibility, bool values "true" and "false" are also allowed for this field.
 # "true" is equivalent to "required" and "false" is equivalent to "off".
 checksum = "required"
+# Specifies whether the ADMIN CHECKSUM TABLE <table> operation is executed via TiDB.
+# The default value is "false", which means that the ADMIN CHECKSUM TABLE <table> command is sent to TiKV for execution via TiDB Lightning.
+# It is recommended that you set this value to "true" to make it easier to locate the problem if checksum fails.
+# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` parameter in TiDB.
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:413]: add the  link for `tidb_checksum_table_concurrency` https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency

Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:413]:
```suggestion
# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` (https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency) parameter in TiDB.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -359,8 +359,8 @@ pd-addr = "172.16.31.4:2379"
 # This setting controls the log level of the TiDB library.
 log-level = "error"
 
-# Sets the TiDB session variable to speed up the Checksum and Analyze operations.
-# See https://pingcap.com/docs/dev/reference/performance/statistics/#control-analyze-concurrency
+# Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", TiDB Lightning will execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency = 15` and `checksum-table-concurrency = 2` will not take effect.
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:362]:
```suggestion
# Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", TiDB Lightning will execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency ` and `checksum-table-concurrency ` will not take effect.
```

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -407,6 +407,11 @@ max-allowed-packet = 67_108_864
 # For backward compatibility, bool values "true" and "false" are also allowed for this field.
 # "true" is equivalent to "required" and "false" is equivalent to "off".
 checksum = "required"
+# Specifies whether the ADMIN CHECKSUM TABLE <table> operation is executed via TiDB.
+# The default value is "false", which means that the ADMIN CHECKSUM TABLE <table> command is sent to TiKV for execution via TiDB Lightning.
+# It is recommended that you set this value to "true" to make it easier to locate the problem if checksum fails.
+# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` (https://docs.pingcap.com/tidb/stable/system-variables#tidb_checksum_table_concurrency) parameter in TiDB.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:413]:
```suggestion
# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` variable in TiDB (https://docs.pingcap.com/tidb/stable/system-variables#tidb_checksum_table_concurrency).
```

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**